data/en/census_household.json

```diff
@@ -3683,22 +3683,20 @@
                 {
                     "type": "Questionnaire",
                     "id": "hours-worked",
-                    "title": "{{[answers.first_name[group_instance], answers.last_name[group_instance]] | format_household_name }}",
                     "description": "",
                     "questions": [{
                         "title": "In your main job, how many hours a week do you usually work?",
+                        "guidance": {
+                            "content": [{
+                                "description": "Include paid and unpaid overtime"
+                            }]
+                        },
                         "id": "hours-worked-question",
                         "type": "General",
                         "answers": [{
                             "id": "hours-worked-answer",
-                            "mandatory": true,
-                            "guidance": {
-                                "show_guidance": "Show further guidance",
-                                "hide_guidance": "Hide further guidance",
-                                "content": [{
-                                    "description": "Include paid and unpaid overtime"
-                                }]
-                            },
+                            "mandatory": false,
```

Comment:
Why were these made non-mandatory?

Reply:
Charlotte said that they were meant to be non mandatory after a discussion with her

Reply:
@charlotte-hirst this isn't documented in the Trello card

Reply:
@jonnyshaw89 double checked with charlotte and its non-mandatory

Reply:
This should be added to the card then as acceptance criteria

Reply:
I've just added them to the cards

Reply:
Does this card also cover changes to the `work-travel` question?

Reply:
Yes covers both I've updated both the cards regarding this below:

hours-worked: https://trello.com/c/KyXiTyqR/1447-add-hours-worked-question-s
travel-to-work: https://trello.com/c/Mg2K0iNL/1445-add-travel-to-work-question-s